client/src/components/reviews/newreview.jsx

```diff
@@ -167,12 +167,12 @@ const NewReview = ({ productID, meta, mainData }) => {
           </div>
         </div>
         <div id='name'>
-          <div className='newInputHeader'>Nickname</div>
+          <div className='newInputHeader'>Nickname *</div>
```

Comment:
What is the purpose of the * ?

Reply:
Just a business requirement to render a * for input fields that are mandatory. I caught this when I read through the doc again.